src/components/FormDatePicker.vue

```diff
@@ -71,25 +74,31 @@
       }
     },
     watch: {
-      dataFormat() {
-        this.updateFormat();
+      dataFormat: {
+        immediate: true,
+        handler() {
+          this.config.format = this.dataFormat === 'datetime' ? this.getUserDateTimeFormat() : this.getUserDateFormat();
+          this.date = this.stdValue(this.value);
+        }
       },
-      value() {
-        this.setDate();
+      value(value) {
+        this.date = this.stdValue(value);
       },
       date() {
         if (typeof this.date === "string") {
-          this.$emit('input', this.date);
+          this.stdValue(this.value) !== this.stdValue(this.date) ? this.$emit('input', this.stdValue(this.date)) : null;
         }
       }
     },
     methods: {
-      updateFormat() {
-        if (this.dataFormat === 'datetime') {
-          this.config.format = 'MM/DD/YYYY h:mm A';
-        } else  {
-          this.config.format = 'MM/DD/YYYY';
-        }
+      getUserDateFormat() {
+        return ProcessMaker.user.datetime_format.replace(/ |H|:|m|s|z|Z/g, '');
```

Comment:
check if there is a variable Processmaker

Reply:
fixed